.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
```

Comment:
I don't like this a lot: someone may wrongly insert an issue with [BUG] as title...
And there's already bug label!

Reply:
I don't think it would happen because github set the focus to it when  a new issue with template BUG is created